File: codigos/2 - estrutura condicional/1035 - Teste de Seleção/Main.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) {return lines.shift();};

function get_validade(A,B,C,D) {
    return B > C && D > A && C+D > A+B && C > 0 && D > 0 && A % 2 == 0;
}

function main(){
    let numeros = prompt().split(' ').map(Number);
    let eh_valido = get_validade(...numeros);
    if (eh_valido) console.log("Valores aceitos");
    else console.log("Valores nao aceitos");
}

main()

File: codigos/2 - estrutura condicional/1035 - Teste de Seleção/Main.py
def validade(A: int, B: int, C: int, D: int) -> bool:
    return B > C and D > A and C+D > A+B and C > 0 and D > 0 and A % 2 == 0

def main() -> None:
    numeros: list[int] = [int(x) for x in input().split(' ')]
    eh_valido: bool = validade(*numeros)
    print("Valores aceitos" if eh_valido else "Valores nao aceitos")
    
if __name__ == "__main__":
    main()